src/components/CompareResults/beta/SearchInput.tsx

```diff
@@ -0,0 +1,57 @@
+import Close from '@mui/icons-material/Close';
+import SearchIcon from '@mui/icons-material/Search';
+import { FormControl, InputAdornment, TextField, Button } from '@mui/material';
+import { style } from 'typestyle';
+
+import { ButtonsLightRaw, Spacing } from '../../../styles';
+
+const styles = {
+  container: style({
+    display: 'flex',
+    $nest: {
+      '.form-container': {
+        maxWidth: '447px',
+      },
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      '.MuiButtonBase-root': {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        ...ButtonsLightRaw.Secondary,
+        marginLeft: Spacing.Small,
+      },
```

Comment:
@Carla-Moz do you have an idea how to avoid these eslint comments in future PRs?

Reply:
Ah, yes, I'll create an issue for that. We probably have to update something in ts config files for these style files. I'll look into it. Maybe exclude the styles directory.

Reply:
@esanuandra can you try updating your npm to latest version and see if that works. What version are you using now?